tests/xharness/Xharness.Tests/Hardware/Tests/SimDeviceTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using Xharness.Execution;
+using Xharness.Hardware;
+using Xharness.Logging;
+
+namespace Xharness.Tests.Hardware.Tests {
+	[TestFixture]
+	public class SImDeviceTest {
```

Comment:
^ alignment

Reply:
not so sure, it looks like a tab that did not use to be 4 spaces
maybe it's an GitHub change (related to .editorconfig) ?

Reply:
it is a tab. I have it setup to 4 and looks as expected. Funny .editorconfig side effect.

Reply:
Looks like a GitHub bug... 2 space indent for the first indentation. Maybe they're counting the `+` and the space after that as indentation?